examples/scripts/ct_multi_cs_tv_admm.py

```diff
@@ -38,41 +38,54 @@
 np.random.seed(1234)
 x_gt = snp.array(discrete_phantom(Foam(size_range=[0.075, 0.0025], gap=1e-3, porosity=1), size=N))
 
+det_count = N
+det_spacing = np.sqrt(2)
 
 """
 Define CT geometry and construct array of (approximately) equivalent projectors.
 """
 n_projection = 45  # number of projections
 angles = np.linspace(0, np.pi, n_projection)  # evenly spaced projection angles
 projectors = {
-    "astra": astra.XRayTransform2D(x_gt.shape, N, 1.0, angles - np.pi / 2.0),  # astra
-    "svmbir": svmbir.XRayTransform(x_gt.shape, 2 * np.pi - angles, N),  # svmbir
-    "scico": XRayTransform(Parallel2dProjector((N, N), angles, det_count=N)),  # scico
+    "astra": astra.XRayTransform2D(
+        x_gt.shape, det_count, det_spacing, angles - np.pi / 2.0
+    ),  # astra
+    "svmbir": svmbir.XRayTransform(
+        x_gt.shape, 2 * np.pi - angles, det_count, delta_pixel=1.0, delta_channel=det_spacing
+    ),  # svmbir
+    "scico": XRayTransform(
+        Parallel2dProjector((N, N), angles, det_count=det_count, dx=1 / det_spacing)
+    ),  # scico
 }
 
-
```

Comment:
Please restore this linefeed: the "standard" for example scripts is two linefeeds between each code "block".